harry-runner/src/harry/runner/HarryRunner.java

```diff
@@ -107,7 +71,8 @@ public void runWithInJvmDtest() throws Throwable
                 return a;
             }).get(run.snapshot.run_time_unit.toSeconds(run.snapshot.run_time) + 30,
                    TimeUnit.SECONDS);
-            ((Throwable) result).printStackTrace();
```

Comment:
this was throwing an NPE on successful execution

Reply:
We can (and probably should) switch to `logger.error` here?

Reply:
Fixed.